eyeshade/controllers/publishers.js

```diff
@@ -149,6 +159,7 @@ module.exports.initialize = async (debug, runtime) => {
         commission: bson.Decimal128.POSITIVE_ZERO,    // conversion fee (i.e., for settlement)
         fee: bson.Decimal128.POSITIVE_ZERO,           // network fee (i.e., for settlement)
         type: '',
+        executedAt: bson.Timestamp.ZERO,              // When the settlement was executed
```

Comment:
should this be `bson.Date.ZERO`?

Reply:
Date input will be turned into and stored as timestamp by mongo